app/controllers/renewals_controller.rb

```diff
@@ -37,7 +38,13 @@ def checkouts_to_renew
     end
 
     def renew_params
-      params.require(:renewal_list)
+      if params[:renewal_list].blank?
+        flash[:notice] = 'No items were selected for renewal.'
+
+        redirect_to checkouts_path
+      else
```

Comment:
I kinda feel like this should be a front-end validate, like unless a checkbox is checked don't allow the button to be clicked (i.e, disable it). What do you think? For the ruby code you can get rid of that else.

```suggestion
      end
      params[:renewal_list]

Reply:
yeah, I want to add a front-end validate for this,that's why I brought up the form validations the other day, did not want to add front-end stuff yet before we talk about it. I do not know if it hurts to keep this check as well?

Reply:
Oh sorry, I forgot that conversation. I can see pros and cons of doing it just front-end, just back-end or both. We can chat off GH of course. For now, just backend is in place and working so let's leave it. 👍

Reply:
yes, let's chat about it and depending on what we decide how we will handle the validations in general, I can refactor this part again or remove etc.